peer.go

```diff
@@ -427,6 +445,22 @@ func (p *peer) loadActiveChannels(chans []*channeldb.OpenChannel) error {
 		case dbChan.HasChanStatus(channeldb.ChanStatusLocalDataLoss):
 			peerLog.Warnf("ChannelPoint(%v) has status %v, won't "+
 				"start.", chanPoint, dbChan.ChanStatus())
+
+			// To help our peer recover from a potential data loss,
+			// we resend our channel reestablish message if the
+			// channel is in a borked state. We won't process any
+			// channel reestablish message sent from the peer, but
+			// that's okay since the assumption is that we did when
+			// marking the channel borked.
+			chanSync, err := dbChan.ChanSyncMsg(false)
```

Comment:
Why `false` here? We don't have enough context to know if it was recovered or not. 

Nvm scratch that.